src/System.Private.Uri/tests/FunctionalTests/IdnDnsSafeHostTest.cs

```diff
@@ -45,11 +45,13 @@ public void IdnDnsSafeHost_IPv6Host_ScopeIdButNoBrackets()
             Assert.Equal("[::1]", test.Host);
         }
 
-        [Theory]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
-        [InlineData("fe80::e077:c9a3:eeba:b8e9", "%18")]
-        public void IdnDnsSafeHost_IPv6HostLinkLocalAddress_ScopeIdCorrectlyFormatted(string address, string zoneIndex)
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]        
+        public void IdnDnsSafeHost_IPv6HostLinkLocalAddress_ScopeIdCorrectlyFormatted()
         {
+            string address = "fe80::e077:c9a3:eeba:b8e9";
+            string zoneIndex = "%18";
+
```

Comment:
Use
```c#
const string ScopedLiteralIpv6 = "fe80::e077:c9a3:eeba:b8e9%18"; 
```
since `address` and `zoneIndex` is not used anywhere else in the test.

Reply:
oh yes i missed it sorry

Reply:
done, updated also other test with constants.